lib/editor-linter.coffee

```diff
@@ -8,8 +8,11 @@ class EditorLinter
     @inProgress = false
     @inProgressFly = false
 
+    @lineForMessages = 0 # It's used to compare if the user has changed line so we can updateLineMessages
```

Comment:
`TextEditor.onDidChangeCursorPosition` has `oldBufferPosition` argument, which I think makes this redundant.